xdsl/tools/command_line_tool.py

```diff
@@ -130,6 +131,7 @@ def get_all_dialects() -> list[Dialect]:
 def get_all_passes() -> list[type[ModulePass]]:
     """Return the list of all available passes."""
     return [
+        gpu_map_parallel_loops.GpuMapParallelLoopsPass,
```

Comment:
can you put it lower on the list, so it stays alphabetical, please?

Reply:
AFAIU, it is not; I'll put at a spot where it is somehow locally in order, sure

Reply:
thank you. At some point I'll open a PR to sort it again :)